### ISSUE TITLE
intro: causal & why?

### ISSUE THREAD

**Issue by nickeubank** [nickeubank/ds4humans#13]
Adriane: 

> Passive prediction doesn't care about "why", right?  Whether you understand why a patient is likely to get cancer or not is irrelevant to your task.  With enough data, you can put anything in your model, you don't even need to think bout whther its a good idea to put it in the model. A good prediction is a good prediction regardless of your understanding.  Causality begins to move you closer to questions of why.  You likely have an inkling of an understanding in order to choose to manipulate a cause, or think about a cause in the first place.

**Comment by joshclinton** [comment]
I Agree.  You can correctly predict using correlations.  That said, the absence of causality makes it likely that your prediction goes awry -- think about the dumb predictions about Washington Football team and presidential elections.  You do not need causality to make a prediction, but causality helps ensure that the prediction is not simply spurious.  We evaluate predictions by the ability to classify correctly (cross-validation, etc.) -- not explain why an outcome occurs.
